xivo_dao/resources/utils/search.py

```diff
@@ -191,7 +191,7 @@ def _apply_search_params(self, rows, order, limit, offset, reverse):
         if order:
             rows = sorted(
                 rows,
-                key=lambda x: unicodedata.normalize('NFKD', getattr(x, order)),
+                key=lambda x: unicodedata.normalize('NFKD', getattr(x, order, '')),
```

Comment:
This covers case where the fields doesn't exist(which should perhaps be treated as an error since you should not try to order by an invalid attribute).
But this doesn't cover the case where the field is nullable and the attribute exists but is None.